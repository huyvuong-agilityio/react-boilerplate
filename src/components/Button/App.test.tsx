// Libraries
import * as renderer from 'react-test-renderer';

// Components
import { Button } from '@components';

describe('App', () => {
  test('It should render correctly', () => {
    const component = renderer.create(<Button>Click</Button>).toJSON();
    expect(component).toMatchSnapshot();
  });
});
