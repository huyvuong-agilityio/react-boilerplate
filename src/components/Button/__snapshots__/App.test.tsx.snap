// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App It should render correctly 1`] = `
<button>
  Click
</button>
`;
